### PR TITLE
Implement "neco reboot-and-wait" that reboots a machine and waits for its start-up.

### PR DESCRIPTION
**Pull request by morimoto-cybozu**
Signed-off-by: morimoto-cybozu <kenji_morimoto@cybozu.co.jp>

### CODE REVIEW

File: pkg/neco/cmd/reboot_and_wait.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+var rebootAndWaitCmd = &cobra.Command{
+	Use:   "reboot-and-wait SERIAL_OR_IP",
+	Short: "reboot a machine and wait for its boot-up",
+	Long:  `Reboot a machine having SERIAL or IP address, and wait for its boot-up.`,
+
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return rebootAndWaitMain(args[0])
```
Comment (ymmt2005) [inline pkg/neco/cmd/reboot_and_wait.go:41]:
```suggestion
		cmd.SilenceUsage = true
		return rebootAndWaitMain(args[0])
```

File: pkg/neco/cmd/reboot_and_wait.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	"encoding/json"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/cybozu-go/log"
+	"github.com/spf13/cobra"
+)
+
+const serfTagUptime = "update"
```
Comment (ymmt2005) [inline pkg/neco/cmd/reboot_and_wait.go:14]: uptime?

Comment (morimoto-cybozu) [reply pkg/neco/cmd/reboot_and_wait.go:14]: Oops, I mistyped "uptime" repeatedly and made it a constant not to mistype any more.
And I mistyped at the most critical time...

Comment (ymmt2005) [reply pkg/neco/cmd/reboot_and_wait.go:14]: You can force-push the fix.  Everything except where I commented looks good.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**